byte REMOTE_CONTROL_ALLOWED=1
byte REMOTE_CONTROL_DENIED=0

byte REMOTE_CONTROL_ENABLED=1
byte REMOTE_CONTROL_DISABLED=0

Header header

uint8 remote_control_allowed
uint8 remote_control_enabled
